### PR TITLE
Add blur at the end of playlist

### PR DESCRIPTION
**Pull request by bobobob22**

### CODE REVIEW

File: packages/webapp/src/components/AddSongs/AddSongs.component.tsx [hunk 1]
```diff
@@ -1,19 +1,52 @@
...
 
 export const AddSongs: FC<IAddSongsProps> = ({ previewSongInfo, fetchedSongs, playSongPreview, addSongToPlaylist }) => {
   const numberOfFetchedSongs = fetchedSongs.length
+  const [isBottom, setIsBottom] = useState(false)
+  const [isTop, setIsTop] = useState(false)
+
+  const songListContainer = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    throttledScroll()
```
Comment (kowalski) [inline packages/webapp/src/components/AddSongs/AddSongs.component.tsx:27]: you are using this before setting it in line 49
I'm suprised linter doesn't object

File: packages/webapp/src/components/AddSongs/AddSongs.component.tsx [hunk 2]
```diff
@@ -1,19 +1,52 @@
...
+  }, [])
```
Comment (kowalski) [inline packages/webapp/src/components/AddSongs/AddSongs.component.tsx:28]: missing dependency

File: packages/webapp/src/components/AddSongs/AddSongs.component.tsx [hunk 3]
```diff
@@ -1,19 +1,52 @@
...
+  const handleScroll = useCallback(() => {
+    if (songListContainer.current && songListContainer.current.scrollTop === 0) {
+      setIsTop(true)
+    } else {
+      setIsTop(false)
+    }
+    if (
+      songListContainer.current &&
+      songListContainer.current.lastElementChild &&
+      songListContainer.current.clientHeight + songListContainer.current.scrollTop ===
```
Comment (kowalski) [inline packages/webapp/src/components/AddSongs/AddSongs.component.tsx:39]: I don't get how this condition could be right, but if it works, than fine

**Review by kowalski — CHANGES_REQUESTED** [review]

File: packages/webapp/src/components/AddSongs/AddSongs.component.tsx (hunk 3 above)
Comment (bobobob22) [reply packages/webapp/src/components/AddSongs/AddSongs.component.tsx:32]: it seems to work

File: packages/webapp/src/components/AddSongs/AddSongs.component.tsx (hunk 2 above)
Comment (bobobob22) [reply packages/webapp/src/components/AddSongs/AddSongs.component.tsx]: ok

File: packages/webapp/src/components/AddSongs/AddSongs.component.tsx (hunk 1 above)
Comment (bobobob22) [reply packages/webapp/src/components/AddSongs/AddSongs.component.tsx]: ok, fixed

**Pull request merged**